packages/react/src/components/Tree/TreeTitle.tsx

```diff
@@ -66,6 +66,22 @@ class TreeTitle extends UIComponent<WithAsProp<TreeTitleProps>> {
       e.preventDefault()
       this.handleClick(e)
     },
+    expandOrFocusFirstChild: e => {
+      const { open } = this.props
+      e.preventDefault()
```

Comment:
just a thought, something we can discuss: these action handlers just call handle click based on the open prop. I would say they have nothing to do with expand/collapse, but tree item does as it is responsible for rendering subtree. I would rather rename these action handlers